<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Z.X., Koong</author>
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 170, 00);
background-color: rgb(255, 246, 249);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QCheckBox" name="istimetrace">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Time trace</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isG2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="filepath">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Analyse PTU</string>
   </property>
  </widget>
  <widget class="QToolButton" name="openfile">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>60</y>
     <width>32</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>451</width>
     <height>291</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Parameters</string>
   </property>
   <widget class="QLineEdit" name="postselect0">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="postselect1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>260</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>214</width>
      <height>220</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Time trace res (s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>G2 time res (ps)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>G2 time window (ns)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Zero Delay Position (ns)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>G2 Algorithm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>G2 Post Selection</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>30</y>
      <width>195</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="timetraceres">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="g2timeres">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="minimum">
        <number>4</number>
       </property>
       <property name="maximum">
        <number>512</number>
       </property>
       <property name="singleStep">
        <number>4</number>
       </property>
       <property name="value">
        <number>32</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="g2timewindow">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zerodelaypos">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(85, 255, 127);
background-color: rgb(255, 246, 249);</string>
       </property>
       <item>
        <property name="text">
         <string>Symmetric</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ring</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="g2postselectionopts">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 246, 249);</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Above</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Below</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Between</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="isSavedData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>470</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Saved Data</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isSaveFigure">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save Figure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CalculateG2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 201, 238, 155);</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="savebutton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>112</width>
     <height>43</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 255);
background-color: rgb(223, 215, 255);</string>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="calculatebar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QProgressBar" name="saveprogress">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>510</y>
     <width>120</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>filepath</tabstop>
  <tabstop>openfile</tabstop>
  <tabstop>istimetrace</tabstop>
  <tabstop>isG2</tabstop>
  <tabstop>timetraceres</tabstop>
  <tabstop>g2timeres</tabstop>
  <tabstop>g2timewindow</tabstop>
  <tabstop>zerodelaypos</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>g2postselectionopts</tabstop>
  <tabstop>postselect0</tabstop>
  <tabstop>postselect1</tabstop>
  <tabstop>isSavedData</tabstop>
  <tabstop>isSaveFigure</tabstop>
  <tabstop>savebutton</tabstop>
  <tabstop>CalculateG2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>10</number>
  </property>
  <property name="gridDeltaY">
   <number>10</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
